src/components/RadioButtonGroup/styled/marker.js

```diff
@@ -32,15 +22,15 @@ const StyledMarker = styled.span.attrs(props => {
     ${props =>
         props.variant === 'inverse' &&
         `
-            box-shadow: 0 0 4px 0 ${props.brandMainColor};
-            background-color: ${props.brandMainColor};
-            border: solid 1px ${props.brandMainColor};
+            box-shadow: 0 0 4px 0 ${props.palette.brand.main};
```

Comment:
@TahimiLeonBravo  check this shadow

Reply:
this shadow was originally box-shadow: 0 0 4px 0 ${props.brandMainColor}; 
it is not on the shadows list..... and in order to avoid adding more... i wasn't sure what to do... so i only changed the color

Reply:
i also changed the shadow for variant="brand".. it had a rgba(...) color so i change it in order to use theme colors... perhaps instead of having entire shadows definitions ... it would be better to have a shadows color section in palette ... like borders and backgrounds sections...